### ISSUE TITLE
Creating access token via ui does not lead back to the application overview

### ISSUE THREAD

**Issue by chriz-uniba** [inveniosoftware/invenio-app-rdm#1143]
Package version (if known): v7 (but also older versions had this issue)

## Describe the bug

When creating an access token the last step does not lead back to the Application overview but shows me a 'save'-button that does nothing? or it does something and the user does not get informed that is does something - you will stay on the same page and can click the button several times with no change. 

## Steps to Reproduce

1. Sign up https://inveniordm.web.cern.ch/signup/
2. Go to 'Applications' (drop down on the signed up account)
3. Create an access token:

- Click here on 'New token':
![01-step](https://user-images.githubusercontent.com/74763324/145226176-e7b619a8-6503-43d4-82b1-1995f40510fc.png)
- Add a name and click 'Create':
![02-step](https://user-images.githubusercontent.com/74763324/145226196-84433b61-e495-4403-8e98-58b33c4e1752.png)
- Click on 'Save'
![03-step](https://user-images.githubusercontent.com/74763324/145226233-aef457a5-beda-4249-9315-2e6cdcd4f39d.png)
- Leads to:
![04-step](https://user-images.githubusercontent.com/74763324/145226239-8f972df9-2fc3-41c1-9fcd-a5b73333ef28.png)

## Expected behavior

I'm not sure whether the 'save' button on the last step leads to something - if it is needed, then after clicking it you might should get back to the 'Applications' overview.

![05-step](https://user-images.githubusercontent.com/74763324/145226668-2878758f-d9c3-4ba0-9fd2-3f76ddcf2214.png)

Currently you have to click on any of the other menu to be able to 'reload' or select again the 'Applications' view.

**Comment by lnielsen** [comment]
The save button allows you to change the scopes (currently only one today) and the name of the token so I think it's correct that it stays on the page.

**Comment by chriz-uniba** [comment]
I disagree. After the creation of a token you are not able to come back to this page in a reasonable way:

![image](https://user-images.githubusercontent.com/74763324/154238931-548b1ba5-07cb-4da1-b724-3fe3051a386e.png)

You stay on the page with the token:

![image](https://user-images.githubusercontent.com/74763324/154239065-249454b6-86a7-4229-92e9-6f4565e0acaf.png)

The only way to come back to the 'Applications'-Overview is by reloading your whole page, or by clicking on some of the other menu (profile, change password, ...) and then click again on 'applications'.

**Comment by lnielsen** [comment]
The current behaviour of save button in other places in Invenio is that it stays on the detail page that you're editing so I don't want to change this part of the behaviour. 

We can either make "Applications" linked or add a third button saying staying e.g. "Cance"l that will lead back.
